Sky Atlas/Sky Atlas.html

```diff
@@ -24712,39 +24759,69 @@ <h1>Agent Scope Database</h1>
           </td>
           <td>Spark</td>
           <td>Core</td>
-          <td>The Root Edit process begins with a SPK token holder submitting a proposal through the Powerhouse system containing a draft Artifact Edit Proposal. A SPK token holder must hold at least 1% of the circulating token supply to submit a proposal. The proposal must also be posted on the Sky Forum under the "Spark" tag.</td>
+          <td><p>The Root Edit process begins with a SPK token holder submitting a proposal through the Powerhouse system containing a draft Artifact Edit Proposal. A SPK token holder must hold at least 1% of the total token supply to submit a proposal. The proposal must also be posted on the Sky Forum under the “Spark” tag.</p></td>
         </tr>
         <tr>
           <td>
-            <dfn>Root Edit Proposal Submission Requirements Exception</dfn>
+            <dfn>Root Edit Proposal Submission Requirements Exception For Nested Contributors</dfn>
+          </td>
+          <td>Spark</td>
+          <td>Core</td>
+          <td><p>Nested Contributors are always authorized to submit Artifact Edit Proposals and do not have to fulfill the token-holding requirements defined in Root Edit Proposal Submission. However, all other procedural requirements within the Root Edit process continue to apply.</p><p>To see the Agent’s Nested Contributors, see <dfn>Phoenix Labs</dfn>.</p></td>
+        </tr>
+        <tr>
+          <td>
+            <dfn>Root Edit Proposal Submission Requirements Exception For Spark Foundation</dfn>
+          </td>
+          <td>Spark</td>
+          <td>Core</td>
+          <td><p>While it is a core service provider to Spark, the Spark Foundation is authorized to submit Artifact Edit Proposals and does not have to fulfill the token-holding requirements defined in Root Edit Proposal Submission. However, all other procedural requirements within the Root Edit process continue to apply.</p></td>
+        </tr>
+        <tr>
+          <td>
+            <dfn>Short-Term Transitionary Measures</dfn>
           </td>
           <td>Spark</td>
           <td>Core</td>
-          <td>For proposals that solely entail a buyback or a grant of SPK tokens, the requirement that SPK token holders must hold at least 1% of the circulating token supply to submit a proposal is waived. However, all other procedural requirements within the Root Edit process continue to apply.</td>
+          <td><p>Until the Powerhouse system supports submitting Artifact Edit Proposals, SPK token holders may submit Artifact Edit Proposals by posting them to the Sky Forum under the “Spark” category. The title of the post must include the text “Spark Artifact Edit Proposal”. The post must include cryptographic proof that the author controls an account holding the required percentage of the total SPK token supply specified in Root Edit Proposal Submission.</p></td>
```

Comment:
1. nit: You don't need p tags for a single line entry
2. Maintain consistency of quotation mark formatting

```suggestion
          <td>Until the Powerhouse system supports submitting Artifact Edit Proposals, SPK token holders may submit Artifact Edit Proposals by posting them to the Sky Forum under the "Spark" category. The title of the post must include the text "Spark Artifact Edit Proposal". The post must include cryptographic proof that the author controls an account holding the required percentage of the total SPK token supply specified in Root Edit Proposal Submission.</td>
```